### PR TITLE
Docs: Add Event log to reporting specification 

### PR DESCRIPTION
**Pull request by tspurway**
This is the spec for the changes proposed by #387

r? @k88hudson @ncloudioj

### CODE REVIEW

**Comment by k88hudson** [comment]
Awesome :+1: I would rename `click_position` to be more general on the events ping definition, but other than that looks good

File: data_dictionary.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
-# Example Activity Stream Log
+# Activity Stream Pings
+
+The Activity Stream addon sends two distinct types of pings to the backend (HTTPS POST) [Onyx server](https://github.com/mozilla/onyx) :
+- a `session` ping that describes the ending of an Ativity Stream session (lose focus event on Activity Stream), and
```
Comment (pdehaan) [inline data_dictionary.md]: typo: Ativity

File: data_dictionary.md [hunk 2]
```diff
@@ -23,14 +28,34 @@
...
+{
+  "action": "activity_stream_event",
+  "addon_version": "1.0.0",
+  "click_position": 4,
+  "client_id": "374dc4d8-0cb2-4ac5-a3cf-c5a9bc3c602e",
+  "date": "2016-03-07",
+  "ip": "10.192.171.13",
+  "locale": "en-US",
+  "source": "top_sites",
+  "event": "delete",
```
Comment (pdehaan) [inline data_dictionary.md]: Bonus points if you nudge `event` up a few lines to preserve the glorious A-Z sorting.

File: data_dictionary.md [hunk 3]
```diff
@@ -23,14 +28,34 @@
...
+  "tab_id": 3,
+  "timestamp": 1457396660000,
+  "ua": "python-requests/2.9.1",
+  "ver": "3"
```
Comment (ncloudioj) [inline data_dictionary.md]: The `ver` is auto-populated by Onyx, the client doesn't need this.

File: data_dictionary.md [hunk 4]
```diff
@@ -23,14 +28,34 @@
...
+  "action": "activity_stream_event",
+  "addon_version": "1.0.0",
+  "click_position": 4,
+  "client_id": "374dc4d8-0cb2-4ac5-a3cf-c5a9bc3c602e",
+  "date": "2016-03-07",
+  "ip": "10.192.171.13",
+  "locale": "en-US",
+  "source": "top_sites",
+  "event": "delete",
+  "tab_id": 3,
```
Comment (ncloudioj) [inline data_dictionary.md]: Each `tab_id` is a string, e.g. "1-3", which stands for a combination of window id and tab id.

**Comment by ncloudioj** [comment]
LGTM @tspurway

**Pull request merged**